eos/observable.cc

```diff
@@ -86,7 +88,11 @@ namespace eos
 
             void insert(const QualifiedName & key, const std::shared_ptr<const ObservableEntry> & value)
             {
-                _entries[key] = value;
+                auto inserted = _entries.insert(std::pair<QualifiedName, std::shared_ptr<const ObservableEntry>>(key, value));
+                if (!inserted.second)
```

Comment:
Coding style: ``! inserted.second``. I wonder if we should rather return the boolean value, rather than ``void`` here, and let the callers of this ``ObservableEntries::insert()`` deal with potential exceptions.